tests/conftest.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+import pytest
+from page_analyzer.utils.db import connect_db
+
+
+@pytest.fixture
+def test_database():
+    """
+    Set environment variable "DATABASE_URL" for test database in setup part,
```

Comment:
wdyt about this variant? 

```suggestion
    Makes tests to use test database

    On setup: sets environment variable "DATABASE_URL" for test database

    On teardown: cleans test database
```